pkg/config/sink.go

```diff
@@ -193,6 +193,12 @@ type SinkConfig struct {
 
 	// Debezium only. Whether schema should be excluded in the output.
 	DebeziumDisableSchema *bool `toml:"debezium-disable-schema" json:"debezium-disable-schema,omitempty"`
+
+	// OpenProtocol related configurations
+	OpenProtocol *OpenProtocolConfig `toml:"open" json:"open,omitempty"`
+
+	// DebeziumConfig related configurations
+	Debezium *DebeziumConfig `toml:"debezium" json:"debezium,omitempty"`
```

Comment:
It's better to move DebeziumDisableSchema into DebeziumConfig.

Reply:
DebeziumDisableSchema is already published, we shouldn't break the compatibility